File: py/multiplication_table.py
# Task
# Create a NxN multiplicate table of size provided in the parameters
# size 3
# 1 2 3
# 2 4 6
# 3 6 9 

def multiplication_table(size):
  fin_array = []
  for x in range(1, size + 1):
    mult_row = []
    for y in range(1, size + 1):
      mult_row.append(x * y)
    fin_array.append(mult_row)
  return fin_array

print(multiplication_table(3))

File: js/multiplication_table.js
// Task
// Return a multiplication table

function multiplicationTable(row,col){
    master_array = []
    push_array = []
    while (master_array.length < row) {
        for (let x = 1; x <= row; x++) {
            for (let y = 1; y <= col; y++) {
                push_array.push(x * y)
                if (y == col) {
                    master_array.push(push_array)
                    push_array = []
                }
            }
        }  
    }

    return master_array  
}

console.log(multiplicationTable(2,2))
console.log(multiplicationTable(3,3))
console.log(multiplicationTable(2,5))